ansible/roles/kafka/tasks/deploy.yml

```diff
@@ -15,7 +15,7 @@
       - "{{ zookeeper.port }}:2181"
 
 - name: wait until the Zookeeper in this host is up and running
-  action: shell echo ruok | nc -w 3 -i 1 {{ inventory_hostname }} {{ zookeeper.port }}
+  action: shell (echo ruok; sleep 1) | nc {{ inventory_hostname }} {{ zookeeper.port }}
```

Comment:
this seems very sketchy to me. the problem that was fixed by adding `-w 3 -i  1` is that netcat did close the connection too early for zookeeper to respond in some cases. the flags make netcat wait 3 seconds before closing the connection and read only one line per second.
how would a local sleep help here?